src/ascend.js

```diff
@@ -0,0 +1,29 @@
+var _curry1 = require('./internal/_curry1');
+
+
+/**
+ * Makes an "ascending" comparator function out of a function that returns a value
+ * that can be compared
+ *
+ * @func
+ * @memberOf R
+ * @since v0.23.0
+ * @category Function
+ * @sig (a -> Number | String) -> (a, b -> Number)
+ * @param {Function} fn A function of arity one that returns a value that can be compared
+ * @return {Function} A Function :: a -> b -> Int that returns `-1` if a < b, `1` if b < a, otherwise `0`
+ * @example
+ *
+ *      var byAge = R.ascend(R.prop('age'));
+ *      var people = [
+ *        // ...
+ *      ];
+ *      var peopleByYoungestFirst = R.sort(byAge, people);
+ */
+module.exports = _curry1(function ascend(fn) {
+  return function(a, b) {
+    var aa = fn(a)
```

Comment:
I think we use semicolons to end lines of code? Did you see somewhere that does not use them?